.travis.yml

```diff
@@ -1,56 +1,40 @@
 addons:
   apt:
+    sources:
+      - pov-wkhtmltopdf
     packages:
       - expect-dev  # provides unbuffer utility
       - python-lxml # because pip installation is slow
+      - poppler-utils # document index
+      - antiword # document index
+      - python-ldap # auth_ldap
+      - wkhtmltopdf # report_webkit
 
 language: python
+sudo: false
 
 python:
   - "2.7"
 
 env:
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE_REGEX="^a"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE_REGEX="^[b-g]"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE_REGEX="^([h-k]|l[^1])"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE_REGEX="^m[a-q]"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE_REGEX="^m[r-z]"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE_REGEX="^[n-o]"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE_REGEX="^p[a-q]"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE_REGEX="^p[r-z]"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE_REGEX="^[q-r]"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE_REGEX="^s"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE_REGEX="^[t-v]"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE_REGEX="^website_[a-m]"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE_REGEX="^website_[n-z]"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE_REGEX="^w([^e]|(e[^b])|(eb[^s])|(ebs[^i])|(ebsi[^t])|(ebsit[^e])|(ebsite[^_]))"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" INCLUDE_REGEX="^[x-z]"
+  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0" INSTALL_OPTIONS="--init base"
```

Comment:
@dreispt something seems to be wrong here: https://travis-ci.org/OCA/OCB/builds/73141859